hack/build.sh

```diff
@@ -92,11 +94,13 @@ source_format() {
      echo "✋   go get golang.org/x/tools/cmd/goimports"
      echo "✋ to enable import cleanup. Import cleanup skipped."
 
-     # Run go fmt insteat
+     # Run go fmt instead
      go_fmt
   else
      echo "🧽 ${S}Format"
-     goimports -w cmd pkg
+     goimports -w $(find -name '*.go' | grep -v vendor)
+     echo "🧽 ${S}Format: Simplify code.."
+     gofmt -s -w $(find -path './vendor' -prune -o -type f -name '*.go' -print)
```

Comment:
```suggestion
     find $(echo $source_dir) -name "*.go" -print0 | xargs gofmt -s -w
```

Reply:
Why are you using `-s` here but not in the variation above ?

Reply:
I thought on that, but as I originally used separate Format message for simplifying code, I used a separate command altogether.

Reply:
With single message for formatting, added `-s` option.